import_export/admin.py

```diff
@@ -741,7 +760,12 @@ def get_export_queryset(self, request):
         changelist_kwargs["sortable_by"] = self.sortable_by
         if django.VERSION >= (4, 0):
             changelist_kwargs["search_help_text"] = self.search_help_text
-        cl = ChangeList(**changelist_kwargs)
+
+        # Temporarily set to False to avoid unnecessary COUNT queries.
+        with temp_attr(self, "show_full_result_count", False):
```

Comment:
I think this is fine, but I'm not sure if it will have any side effects.  How about we wrap this block in a setting called `IMPORT_EXPORT_DISABLE_EXPORT_COUNT`, then we can document the reasons for this, and users can opt into disabling the check if they like.